job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/wrapper/SauceOnDemandContext.groovy

```diff
@@ -0,0 +1,89 @@
+package javaposse.jobdsl.dsl.helpers.wrapper
+
+import javaposse.jobdsl.dsl.Context
+
+class SauceOnDemandContext implements Context {
+
+    boolean useGeneratedTunnelIdentifier = false
+    boolean sendUsageData = false
+    boolean useChromeForAndroid = false
+    boolean useOldSauceConnect = false
+    boolean enableSauceConnect = true
+    boolean launchSauceConnectOnSlave = false
+    boolean useLatestVersion = false
+    boolean verboseLogging = false
+    String nativeAppPackage,
+           sauceConnectPath,
+           seleniumHost,
+           seleniumPort,
+           httpsProtocol,
+           options
+
+    List<String> webDriverBrowsers=[]
+    List<String> appiumBrowsers=[]
+
+    void appiumBrowsers(String... appiumBrowsers) {
+        this.appiumBrowsers.addAll(appiumBrowsers)
+    }
+
+    void webDriverBrowsers(String... webDriverBrowsers) {
+        this.webDriverBrowsers = webDriverBrowsers
+    }
+
+    void options(String options) {
+        this.options = options
+    }
+
+    void httpsProtocol(String httpsProtocol) {
+        this.httpsProtocol = httpsProtocol
+    }
+
+    void seleniumPort(String seleniumPort) {
+        this.seleniumPort = seleniumPort
+    }
+
+    void seleniumHost(String seleniumHost) {
+        this.seleniumHost = seleniumHost
+    }
+
+    void sauceConnectPath(String sauceConnectPath) {
+        this.sauceConnectPath = sauceConnectPath
+    }
+
+    void nativeAppPackage(String nativeAppPackage) {
+        this.nativeAppPackage = nativeAppPackage
+    }
+
+    void verboseLogging(boolean verboseLogging) {
```

Comment:
Add a default value of `true` for all boolean parameters, e.g. `void verboseLogging(boolean verboseLogging = true) { ... }`